chaincode-labs/chaincode-seminars/lightning-network/2019-07-22-rene-pickhardt-onion-routing-with-htlcs.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Rene Pickhardt - Onion Routing with HTLCs on the Lightning Network 
+Transcript_by: Dulce Villarreal
+categories: ['chaincode-seminars']
+tags: ['lightning', 'onion routing']
+---
+
+
+Name: Rene Pickhardt
+
+Topic: Onion Routing with HTLCs on the Lightning Network 
+
+Location: Part of “Hacking the Lightning Network - A protocol to scale Bitcoin” book project
+
+Date: Jan 22, 2019
+
+Video: <https://www.youtube.com/watch?v=toarjBSPFqI>
+
+Transcript by: Dulce Villarreal 
+
+
+## Introduction
```

Comment:
Having only this header kind of indicates that this a non-stop Introduction. Maybe consider adding more sections or have no header at all?